### PR TITLE
Add Page class (Frontend code)

### PR DESCRIPTION
**Pull request by nupplaphil**
- Move frontend logic (stylesheet, footerscript, content) to Page class

Another preparation for L10n and Session refactoring.
=> This is less invasive than #7502 

ToDo:
- [x] local node worked
- [x] tests working
- [x] running branch 12h on https://friendica.philpp.info (fatal/error logging on!)

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -280,25 +260,13 @@ public function __construct(Database $database, Configuration $config, App\Mode
 		$this->argc         = $args->getArgc();
 		$this->query_string = $args->getQueryString();
 		$this->module       = $module->getName();
+		$this->page = $page;
```
Comment (MrPetovan) [inline src/App.php]: Please align the equal sign with the others.

**Comment by nupplaphil** [comment]
The `Page` class is implementing the `ArrayAccess` Interface, so for `App` the `page` is still an public array field :) and it maps it on the internal `Page->page` array.

**Comment by MrPetovan** [comment]
Snazzy!

**Comment by nupplaphil** [comment]
Update:
- [x] Installation is still working
- [x] Branch has no failures on my node (checked worker/daemon log too this time ..)

**Pull request merged**